map/lanelet2_extension/include/lanelet2_extension/utility/query.hpp

```diff
@@ -45,6 +47,11 @@ namespace lanelet
 {
 namespace utils
 {
+inline double getAngleDifference(const double angle1, const double angle2)
+{
+  const double diff_angle = std::abs(tier4_autoware_utils::normalizeRadian(angle1 - angle2));
+  return diff_angle;
```

Comment:
I feel there is no need to create a local variable.

Reply:
3b56ce7